docs/how_to_add_packages.md

```diff
@@ -198,7 +216,7 @@ Call `conan create . lib/1.0@` in the folder of the recipe using the profile you
 
 ```sh
 cd conan-center-index/recipes/boost/all
-conan create . boost/1.74.0@
+conan create conanfile.py boost/1.77.0@
```

Comment:
Why `conanfile.py` instead of `.`? Seems unneccesary long

Reply:
To my knowledge Conan V2 will not take directories anymore.. some commands in V1 are already like that

Reply:
true, Conan V2 will be more explicit